### PR TITLE
Update Groupon.com.xml

### PR DESCRIPTION
**Pull request by koops76**
#10782

### CODE REVIEW

File: src/chrome/content/rules/Groupon.com.xml [hunk 1]
```diff
@@ -18,25 +11,23 @@ Fetch error: http://touch2.groupon.com/ => https://touch2.groupon.com/: (28, 'Co
 
 	Nonfunctional subdomains:
 
+		- @ ⁴
+		- www ⁴
```
Comment (J0WI) [inline src/chrome/content/rules/Groupon.com.xml]: I can not confirm the 403 on them. Maybe you have to use a static resource as test url.

**Comment by J0WI** [comment]
This `403` error is likely because of a bug in our test infrastructure.

File: src/chrome/content/rules/Groupon.com.xml [hunk 2]
```diff
@@ -48,25 +36,20 @@ Fetch error: http://touch2.groupon.com/ => https://touch2.groupon.com/: (28, 'Co
...
 	<target host="groupon.com" />
+	<target host="www.groupon.com" />
 	<target host="api.groupon.com" />
 	<target host="api-latam.groupon.com" />
-	<target host="cmdata.groupon.com" />
 	<target host="engineering.groupon.com" />
 	<target host="grassroots.groupon.com" />
 	<target host="jobs.groupon.com" />
-	<target host="m.groupon.com" />
-	<target host="merchants.groupon.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Groupon.com.xml]: WFM

File: src/chrome/content/rules/Groupon.com.xml [hunk 3]
```diff
@@ -25,18 +18,13 @@ Fetch error: http://touch2.groupon.com/ => https://touch2.groupon.com/: (28, 'Co
 
 	Fully covered subdomains:
```
Comment (J0WI) [inline src/chrome/content/rules/Groupon.com.xml]: Can be removed anyway, beacuse all targets ar listed below.

**Comment by J0WI** [comment]
ping @koops76

**Comment by cschanaj** [comment]
@koops76 do you mind to update this PR for completion of #10782?

**Comment by koops76** [comment]
@cschanaj @J0WI What should I change?

**Comment by J0WI** [comment]
See https://github.com/EFForg/https-everywhere/pull/10836#discussion_r128899189 and https://github.com/EFForg/https-everywhere/pull/10836#discussion_r128899217

**Comment by koops76** [comment]
@J0WI Done.

File: src/chrome/content/rules/Groupon.com.xml (hunk 2 above)
Comment (J0WI) [inline src/chrome/content/rules/Groupon.com.xml]: `merchants` WFM, but it seems to fail for Travis :(

**Comment by koops76** [comment]
@J0WI Whitelist I guess?

**Comment by J0WI** [comment]
Let's try a static ressource first:
`http://merchants.groupon.com/assets/main_app/background.png`

**Comment by koops76** [comment]
@J0WI `m` still fails the test.

**Comment by J0WI** [comment]
Yes, my comment was only for `merchants`. Sorry for the confusion.

**Comment by J0WI** [comment]
> Let's try a static ressource first:
`http://merchants.groupon.com/assets/main_app/background.png`

Was this not successful?

**Comment by koops76** [comment]
@J0WI It was, but `m` is still failing the test.

**Comment by J0WI** [comment]
Yes, that's expected since `m` times out. But you also re-removed `merchants`.
I was asking you to try a static resource on `merchants` and otherwise we have to white listing it.

**Comment by koops76** [comment]
@J0WI Done.

**Comment by J0WI** [comment]
Okay, this really requires to be  whitelisted.

**Comment by koops76** [comment]
@J0WI Done.

**Comment by J0WI** [comment]
Thanks!
Please avoid merge commits in PRs.

**Pull request merged**